### PR TITLE
crud: support schema opts

### PR DESCRIPTION
**Pull request by DifferentialOrange**
Support missing `crud.schema` options. Clean up some other crud patches mistakes.

Follows #336, #350

### CODE REVIEW

File: crud/tarantool_test.go [hunk 1]
```diff
@@ -87,6 +87,11 @@ var opObjManyOpts = crud.OperationObjectManyOpts{
 	Timeout: crud.MakeOptFloat64(timeout),
```
Comment (oleg-jukovec) [inline crud/tarantool_test.go:87]: Please, update the `requrests_test.go` too.

Comment (DifferentialOrange) [reply crud/tarantool_test.go:87]: Updated

File: crud/request_test.go [hunk 2]
```diff
@@ -624,6 +634,12 @@ func TestRequestsSetters(t *testing.T) {
 				[]interface{}{spaceName}),
 			target: crud.MakeStatsRequest().Space(spaceName),
 		},
+		{
+			name: "SchemaRequest",
+			ref: tarantool.NewCall17Request("crud.schema").Args(
+				[]interface{}{spaceName}),
+			target: crud.MakeSchemaRequest().Space(spaceName),
```
Comment (oleg-jukovec) [inline crud/request_test.go:641]:
```suggestion
			target: crud.MakeSchemaRequest().Space(spaceName).Opts(schemaOpts),
```

Comment (DifferentialOrange) [reply crud/request_test.go]: Added

File: crud/tarantool_test.go [hunk 3]
```diff
@@ -87,6 +87,11 @@ var opObjManyOpts = crud.OperationObjectManyOpts{
...
 }
 
+var schemaOpts = crud.SchemaOpts{
+	Timeout: crud.MakeOptFloat64(timeout),
+	Cached:  crud.MakeOptBool(false),
```
Comment (oleg-jukovec) [inline crud/tarantool_test.go:92]:
```suggestion
	Timeout: crud.MakeOptFloat64(timeout),
	VshardRouter: crud.MakeOptString("foo"),
	Cached:  crud.MakeOptBool(false),
```

**Review by oleg-jukovec — APPROVED** [review]
Thank you for the patch!

File: crud/tarantool_test.go (hunk 3 above)
Comment (DifferentialOrange) [reply crud/tarantool_test.go:92]: There are integration tests which will start to fail due to unknown Cartridge vshard group name

Comment (DifferentialOrange) [reply crud/tarantool_test.go:92]: Preparing proper tests for vshard group is rather painful (it seems to be the reason why other requests also don't test it now).

Comment (oleg-jukovec) [reply crud/tarantool_test.go:92]: You could make a separate SchemaOpts for the request unit-tests. I'm ok not to test it in an integration test.

**Review by DerekBum — APPROVED** [review]

File: crud/tarantool_test.go (hunk 3 above)
Comment (DifferentialOrange) [reply crud/tarantool_test.go:92]: Added a separate unit test suite in a separate commit.

(I had also discovered that I had forgot to actually encode opts, now it's fixed.)

**Pull request merged**